test/transformers/PAYETransformerTest.scala

```diff
@@ -0,0 +1,449 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package transformers
+
+import org.scalatestplus.play.guice.GuiceOneAppPerTest
+import play.api.libs.json._
+import transformers.PAYETransformer._
+import uk.gov.hmrc.play.test.UnitSpec
+
+import scala.io.Source
+
+class PAYETransformerTest extends UnitSpec with PAYETransformer with GuiceOneAppPerTest {
+
+  val nino = "AB654321B"
+  val payeJson = Json.parse(Source.fromURL(getClass.getResource("/nino_2018.json")).mkString).as[JsObject]
+  lazy val href = s"${PAYETransformer.serviceUrl}/individuals/annual-tax-summary/AB654321B/2018"
+
```

Comment:
extract out the year to a val, it's reused as a magic number throughout this test